posthog/warehouse/api/external_data_schema.py

```diff
@@ -74,3 +83,27 @@ def get_queryset(self):
             return self.queryset.filter(team_id=self.team_id).prefetch_related("created_by").order_by(self.ordering)
 
         return self.queryset.filter(team_id=self.team_id).prefetch_related("created_by").order_by(self.ordering)
+
+    @action(methods=["POST"], detail=True)
+    def reload(self, request: Request, *args: Any, **kwargs: Any):
```

Comment:
If the source is still in legacy format where the schedule per schema hasn't been created, maybe worth adding logic here that creates the schedule?

Reply:
Can this still be the case? All sources would have synced and have had new schedules created since last Weds. Or is there a case when this doesn't hold true?

Reply:
Yeah I was thinking about edge cases where failed jobs could have not rerun properly but it should be small enough to not worry about